site/.build.js

```diff
@@ -4,7 +4,9 @@ const module_path = path.join(__dirname, 'ts');
 
 function getAllFiles(dir) {
     return fs.readdirSync(dir, { withFileTypes: true }).reduce((acc, entry) => {
-        if (!entry.isDirectory() && entry.name.endsWith('.ts') || entry.name.endsWith('.js')) {
+        if (entry.isDirectory()) {
+            acc.push(...getAllFiles(path.join(dir, entry.name)));
```

Comment:
Why do we need to be entering directories? Our entry points are only at the top level of the `ts` folder.

Reply:
@DefinitiveAbove Here is a blurb talking about the TS folder structure https://submitty.org/developer/developing_the_php_site/javascript#creating-new-modules.

The idea is that entry points (top level files) are bundled and will have a corresponding `js` file in the `mjs` directory, and any files in a subdirectory of `ts` will use the es6 module syntax `import` and webpack will just bundle everything together into a single file instead of needing to send out multiple